cpp/ql/src/codeql-suites/cpp-lgtm-full.qls

```diff
@@ -0,0 +1,14 @@
+- description: Standard LGTM queries for C/C++, including ones not displayed by default
+- qlpack: codeql-cpp
+- apply: lgtm-selectors.yml
+  from: codeql-suite-helpers
+# These queries are infeasible to compute on large projects:
+- exclude:
+    dxquery path:
```

Comment:
`dxquery path`? Typo? Is this what it should be?

```suggestion
    - query path:
```

Reply:
Whoops, that was a "disable this condition temporarily for testing" thing that somehow got committed.

Reply:
(It should be just `query path` without the dash, btw).